azure-pipelines.yml

```diff
@@ -14,12 +14,12 @@ trigger:
 
```

Comment:
In the Azure pipeline, it isn't clear to me why line 66 (`pytest --pyargs hyperspy`) does not work if one simply uncomments it.
From the current setup files, it seems that the `dev` dependencies (e.g. pytest, coverage...) are not installed.

Maybe it should be clarified in the README file.

Reply:
Do you have an example of an build which shows this?

Reply:
[Here](https://dev.azure.com/Lumispy/lumispy/_build/results?buildId=17&view=logs&j=bd5e81e7-3dad-5701-f26e-86df05b1109f&t=e8027d99-cc37-535b-4c66-9e7c99e58a70) is an example. 
This is the failed build where the package was simply installed as `pip install .`. 

In the next successful build [here](https://dev.azure.com/Lumispy/lumispy/_build/results?buildId=19&view=results), the tests argument was added in the pip install as

 `pip install .[tests]`

Reply:
Having read your notes on the lumispy [CI issue](https://github.com/LumiSpy/lumispy/issues/24), I guess the reason I had to manually add the `[tests]` in the pip install was because I didn't have the `conda_environment.yml` and `conda_environment_dev.yml` files. I will add those and see if this note is still valid or not.

Reply:
UPDATE:
So I have followed the explanation of using Travis for pip and Azure for conda (which means installing though the `conda_environment.yml` files and now everything makes more sense and it's working without the need to the `[tests]`.

Therefore, ignore this conversation but I would clarify that in the README (especially the clarification of using Travis and Azure for different means).